test/data/cmumps2fhir-patient7.shex

```diff
@@ -0,0 +1,587 @@
+﻿PREFIX fhir:<http://hl7.org/fhir/> 
```

Comment:
No.  This schema was generated by picking up the patient.shex from the FHIR Website and hand crafting the values from the cmumps-patient7.shex.  There is no tool that will generate this automatically as far as I know.  Eric's tools let us validate the schema against a dataset, and let us run that validated dataset against a translation schema like this one to generate RDF. But the generation of both the source schema and the translation schema is manual and requires a fair bit of debugging/testing to get something that works, plus needs extensions to do some of the mappings we need.